src/i18n_check/cli/version.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+"""
+Functions for checking current version of the i18n-check CLI.
+"""
+
+from importlib import metadata
+from typing import Any, Dict
+
+import requests
+
+
+def get_local_version() -> str:
```

Comment:
Could I ask you to add in some doc strings to the new functions in the same style that we have them in the utils, @OmarAI2003? Would be great to get some numpy doc style docstrings in with the base functionality!